release-engineering/role-handbooks/branch-manager.md

```diff
@@ -535,6 +535,12 @@ Proceed with the [alpha release steps](#alpha-releases).
 
 Example `1.18.0-alpha.1` release issue template: https://github.com/kubernetes/sig-release/issues/928
 
+#### Branching perf-tests
+
+Reach out to the sig-scalability folks to ensure a new branch is cut in [sigs.k8s.io/perf-tests](https://github.com/kubernetes/perf-tests/) repository. 
```

Comment:
Is there a recommended way to reach out to them, e.g. is it better to use Slack of send an email? Also, a bit nitty, but I'm wondering if we should be more formal, i.e. say something like `Reach out to SIG Scalability` (dropping folks).

Reply:
> Is there a recommended way to reach out to them, e.g. is it better to use Slack of send an email?

We missed adding information on :point_up: part . How we have done the "reaching out to SIG Scalability" in past?
Could we add links/pointers to that?